tests/integration_tests.rs

```diff
@@ -174,12 +174,17 @@ async fn test_wasi_provider() -> Result<(), Box<dyn std::error::Error>> {
     );
 
     let mut watcher = inf.poll().await?.boxed();
-
+    let mut found_running = false;
     while let Some(event) = watcher.try_next().await? {
         match event {
             WatchEvent::Modified(o) => {
                 let phase = o.status.unwrap().phase.unwrap();
                 if phase == "Running" {
+                    found_running = true;
+                }
+                if phase == "Succeeded" && !found_running {
+                    panic!("Reached completed phase before receiving Running phase")
```

Comment:
could there ever be a case where we see the completed phase before the running phase from the events system, or is the order guaranteed?

Reply:
I think eventing order is guaranteed. This was mostly to check that we actually got a running status reported at some point